internal/container/apikey.go

```diff
@@ -4,14 +4,23 @@ import (
 	"github.com/bitmaelum/bitmaelum-suite/internal/apikey"
 	"github.com/bitmaelum/bitmaelum-suite/internal/config"
 	"github.com/go-redis/redis/v8"
+	"github.com/mitchellh/go-homedir"
 )
 
 // GetAPIKeyRepo returns the repository for storing and fetching api keys
 func GetAPIKeyRepo() apikey.Repository {
-	opts := redis.Options{
-		Addr: config.Server.Redis.Host,
-		DB:   config.Server.Redis.Db,
+	//If redis.host is set on the config file it will use redis instead of bolt
+	if config.Server.Redis.Host != "" {
+		opts := redis.Options{
+			Addr: config.Server.Redis.Host,
+			DB:   config.Server.Redis.Db,
+		}
+
+		return apikey.NewRedisRepository(&opts)
 	}
 
-	return apikey.NewRepository(&opts)
+	//If redis is not set then it will use BoltDB as default
+	dbPath, _ := homedir.Expand(config.Server.Bolt.DatabasePath)
```

Comment:
server settings that need homedir expansion is done during reading of the configuration

Reply:
see: https://github.com/bitmaelum/bitmaelum-suite/blob/develop/internal/config/config_server.go#L97

Reply:
That makes more sense ;)